docs/start.md

```diff
@@ -1,12 +1,10 @@
-# Start Class Proposal
+# Start Class Proposal 
 
-## Starting
-
-Possible to give commandline arguments:
+## Starting Possible to give commandline arguments: 
 
 |command            |effect                                             |
 | ------------------|:-------------------------------------------------:|
-|-isHost            |starts a main Container                            |
+|-isHost {port}     |starts a main Container                            |
```

Comment:
I think this should be ip address instead of port

Reply:
Fixed.